### ISSUE TITLE
Bug that my preset always "converts" to MOV instead of mp4

### ISSUE THREAD

**Issue by bjspi** [paulpacifico/shutter-encoder#236]
Dear Paul & Devs,

I have saved a preset with a few settings about video target bitrate (CQ27 but max bitrate 6MBits), keeping folder-structure/hierarchy and keeping metadata - as I want to convert & process a lot of MOV & mp4 files, all being converted to a MP4.

I stored the preset with selecting "mp4" as target format under "select function" - but when loading the preset, it always resets to "mov" as target file format.

![image](https://github.com/user-attachments/assets/9337a4f6-5cc0-48f4-b21e-657f4b02c48c)

Using latest v18.5 unedr Windows 11 x64.
Can you confirm this "bug" or is it something with my installation

Thanks!

**Comment by paulpacifico** [comment]
Hello,

Did you change the audio codec?

Paul.

**Comment by bjspi** [comment]
Yeah, I noticed that my preset was wrong.
I have source folder with MOV and MP4 files in it which I wanted to convert to mp4 with x265 re-encoding.
So most probably because of dragging MOV+MP4 into the Batch sourcefiles panel, I ended up with a preset with x265 + PCM audio, which was not intended and not noticed by me until recently.

I created a new preset now with AC3 audio and now it stays with the desired file extension and doesnt switch back to MOV.
